src/test/java/seedu/address/model/group/GroupTest.java

```diff
@@ -49,26 +50,29 @@ public void toStringMethod() {
 
     @Test
     public void equals() {
-        // same values -> returns true
-        Group groupCopy = new Group(groupName.toString(), members);
-        assertTrue(group.equals(groupCopy));
+        List<String> studentNames = Arrays.asList("Alice", "Bob");
+        GroupCommand groupCommand1 = new GroupCommand("Group1", studentNames);
+        GroupCommand groupCommand2 = new GroupCommand("Group1", studentNames);
```

Comment:
I don't really agree with the naming convention used here, maybe can consider using the names, groupCommand, and otherGroupCommand instead?